bundles/python/python_lexer.moon

```diff
@@ -79,7 +79,30 @@ howl.aux.lpeg_lexer ->
   raw_string = c('special', S'rR') * basic_string
   encoded_string = c('special', S'bBuU') * any { raw_string, basic_string }
 
-  string = any { basic_string, raw_string, encoded_string }
+  f_prefix = c 'special', any {
+    S'rR' * S'fF'
+    S'fF' * S'rR'
+    S'fF'
+  }
+
+  format_conv = c('operator', P'!') * c('special', S'sra')
+  format_number = any {
+    c 'number', integer
+    c('operator', P'{') * ((V'all' + space + P 1) - P'}')^0 * c('operator', P'}')
```

Comment:
The inner interpolation may not always evaluate to a number - could be any string. It might make sense to move this line to the `format_part` rule below? I don't think it makes any difference whatsoever to the parsing.